patches/tModLoader/Terraria.ModLoader/GlobalTile.cs

```diff
@@ -143,7 +143,7 @@ public class GlobalTile
 		}
 
 		/// <summary>
-		/// Allows you to determine what happens when the tile at the given coordinates is killed or hit with a pickaxe. Fail determines whether the tile is mined, effectOnly makes it so that only dust is created, and noItem stops items from dropping.
+		/// Allows you to determine what happens when the tile at the given coordinates is killed or hit with a pickaxe. fail determines whether the tile is mined. Being true means not mined, effectOnly makes it so that only dust is created, and noItem stops items from dropping.
```

Comment:
```suggestion
		/// Allows you to determine what happens when the tile at the given coordinates is killed or hit with a pickaxe. If <paramref name="fail"/> is true, the tile will not be mined; <paramref name="effectsOnly"/> makes it so that only dust is created; <paramref name="noItem"/> stops items from dropping.
```